website/docs/presets.md

```diff
@@ -112,6 +112,27 @@ module.exports = {
 };
 ```
 
+In addition to these plugins and themes, `@docusaurus/theme-classic` adds [`remark-admonitions`](https://github.com/elviswolcott/remark-admonitions) as a remark plugin to `@docusaurus/plugin-content-blog` and `@docusaurus/plugin-content-docs`.
+
+The `admonitions` key will be passed as the options to `remark-admonitions`. Passing `false` will prevent the plugin from being added to MDX.
+
+```js
+// docusaurus.config.js
+module.exports = {
+  presets: [
+    [
+      '@docusaurus/preset-classic',
+      {
+        docs: {
+          // options for remark-admonitions
+          admonitions: {},
```

Comment:
Could you link to somewhere we can find the docs for remark-admonitions, perhaps a GitHub link?

Reply:
added in b34038a